decidim/decidim
# Improve the bug report template with GitHub form schema

#### :tophat: What? Why?
While working on the [WYSIWYG editor](https://github.com/decidim/decidim/pull/10196) I needed to report some bugs to the Tiptap repository and noticed they are using the GitHub's form schema for their bug reports.

This improves the reporting process by splitting the reportable issues into their own form fields making it a nicer experience to report bugs.

I **propose** this same change also to the Decidim repository.

For further information, see:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

#### Testing
- Go to the Tiptap repository's issues at https://github.com/ueberdosis/tiptap/issues
- Click "New issue"
- Select "Bug report"
- See improvements
